explorer/interpreter/action.cpp

```diff
@@ -44,32 +46,38 @@ void RuntimeScope::Print(llvm::raw_ostream& out) const {
   out << "}";
 }
 
-void RuntimeScope::Bind(ValueNodeView value_node, Nonnull<const Value*> value) {
+void RuntimeScope::Bind(ValueNodeView value_node, Address address) {
+  CARBON_CHECK(!value_node.constant_value().has_value());
+  auto [it, success] =
+      locals_.insert({value_node, heap_->arena().New<LocationValue>(address)});
```

Comment:
```suggestion
  bool success =
      locals_.insert({value_node, heap_->arena().New<LocationValue>(address)}).second;
```

`it` is unused, so no reason to keep it.

Reply:
Thanks, had it fixed in the dependent, but not here. I have a slight preference for `auto [_, success]` vs appending a member access. Any objections?

Reply:
In code, we have cases of `insert(...).second` in resolve_names.cpp and bindings.cpp in the explorer, plus some in the toolchain. I don't think we have any `[_, success]` uses. We seem not to use `_` in bindings in general: I see one spot I did `[param_name_id, _]` in lowering. I think this is actually partly because zygoloid had me change some similar cases.

Consider this a somewhat strong leaning towards `.first`/`.second` over `_` captures for somewhat idiomatic consistency, and because the `.second` still makes it clear there's part of a return value being discarded.

Reply:
Sounds good! Done